auth/user_mgt.go

```diff
@@ -37,6 +39,16 @@ var commonValidators = map[string]func(interface{}) error{
 	"localId":     validateUID,
 }
 
+// Create a new interface
+type relayingpartyCall interface {
```

Comment:
Recommend calling this `identitytoolkitCall`

Reply:
not easy to find the right name :)